pints/toy/_eight_schools_centered.py

```diff
@@ -0,0 +1,96 @@
+#
+# Eight schools log-pdf.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import pints
+
+from . import ToyLogPDF
+
+
+class EightSchoolsCenteredLogPDF(ToyLogPDF):
+    r"""
+    The classic Eight Schools example from [1]_. This model was used to
```

Comment:
Should we be careful using the word 'model' here? To show the implementation is of a model + pdf?

Reply:
I personally think it's ok since model is used by lots of people all over the place. It also really _is_ a model (albeit a statistical model of school performance on SATs); haha.